### ISSUE TITLE
density_from_Universe + exporting to DX file

### ISSUE THREAD

**Issue by casawa** [MDAnalysis/mdanalysis#544]
I'm running the following:

```python
u = Universe(PDB, NC file)
u.trajectory.skip = 1
u.trajectory.skip_timestep = 1
u.trajectory.delta = 1
D = density_from_Universe(u, delta=1.0, atomselection="name OH2")
D.export(DX_FILE_NAME)
```

Then, while exporting to a DX file, I get the following error: `NotImplementedError: Only regularly spaced grids allowed.`

I set delta to be 1.0, so I was confused why this wouldn't be regularly spaced? Thanks!

**Comment by orbeckst** [comment]
Possibly related to #410 .

**Comment by orbeckst** [comment]
There's something weird going on with the OpenDX exporter in `Grid` (see link above). I am working on a PR with MDAnalysis/GridDataFormats#22 but it's work in progress...

**Comment by orbeckst** [comment]
I also started a branch [issue/544](https://github.com/MDAnalysis/mdanalysis/tree/issue/544) with an initial test but we first need to resolve  MDAnalysis/GridDataFormats#22 and publish a fixed 0.3.3 release of GridDataFormats.

**Comment by orbeckst** [comment]
Btw, I assigned the issue to me but of course I am very grateful for any additional help :-)

**Comment by orbeckst** [comment]
Started a PR #563 to monitor progress (especially via tests on travis).

**Comment by orbeckst** [comment]
I just pushed the 0.3.1 release of GridDataFormats. Need to update requirements in MDA to 0.3.1.

**Comment by orbeckst** [comment]
@casawa , the good news is that with the new release of GridDataFormats, it should work for you: Try
```
pip install --upgrade GridDataFormats
```

I will keep this issue open, however, until I can properly test this resolution with our continuous integration (Travis) framework. The whole issue is now really only about this technical problem. *Your* problem should have been solved with the new GridDataFormats release. Please let us know if this works or if you still have problems.

**Comment by orbeckst** [comment]
Closed by PR #563.
